aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -14,35 +14,45 @@
 //! an AWS service client.
 //!
 //! # Examples
-//! Load default SDK configuration:
-//! ```no_run
-//! # mod aws_sdk_dynamodb {
-//! #   pub struct Client;
-//! #   impl Client {
-//! #     pub fn new(config: &aws_types::config::Config) -> Self { Client }
-//! #   }
-//! # }
-//! # async fn docs() {
-//! let config = aws_config::load_from_env().await;
-//! let client = aws_sdk_dynamodb::Client::new(&config);
-//! # }
-//! ```
 //!
-//! Load SDK configuration with a region override:
-//! ```no_run
-//! use aws_config::meta::region::RegionProviderChain;
-//! # mod aws_sdk_dynamodb {
-//! #   pub struct Client;
-//! #   impl Client {
-//! #     pub fn new(config: &aws_types::config::Config) -> Self { Client }
-//! #   }
-//! # }
-//! # async fn docs() {
-//! let region_provider = RegionProviderChain::default_provider().or_else("us-east-1");
-//! let config = aws_config::from_env().region(region_provider).load().await;
-//! let client = aws_sdk_dynamodb::Client::new(&config);
-//! # }
-//! ```
+#![cfg_attr(
```

Comment:
rather than doing this, you can add a hidden feature gate that wraps the example code

Reply:
Clever!